src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The `getStatistic` method is handling multiple responsibilities, making it too long and complex. Consider refactoring the code to separate concerns, which will improve readability and maintainability.